bin/deploy-vhost

```diff
@@ -918,8 +918,9 @@ sub make_dot_forward {
     my $forward_file_name = defined($suffix) ? ".forward-$suffix" : ".forward";
 
     # remove any backup files created by the disable-email script.
-    my $backup_file = glob("~${email_user}/${forward_file_name}.bak");
-    unlink $backup_file if -e $backup_file;
+    if ( -e "/home/${email_user}/${forward_file_name}.bak" ) {
```

Comment:
Doesn't even need this existence check; unlink returns false and sets $! on failure, so you can just ignore it :)